spec/lib/va_profile/prefill/military_information_spec.rb

```diff
@@ -0,0 +1,364 @@
+
+            expect(response).to eq(true)
+          end
+        end
+      end
+
+      describe '#compensable_va_service_connected' do
+        it 'returns false if the rating percentage is not considered "low"' do
+          VCR.use_cassette('va_profile/disability/disability_rating_200_high_disability') do
+            response = subject.compensable_va_service_connected
+
+            expect(response).to eq(false)
+          end
+        end
+      end
+
+      describe '#va_compensation_type' do
+        it "returns 'highDisability' when veteran is paid for a highDisbility" do
+          VCR.use_cassette('va_profile/disability/disability_rating_200_high_disability') do
+            response = subject.va_compensation_type
+
+            expect(response).to eq('highDisability')
+          end
+        end
+      end
+    end
+
+    context 'using bio path disabilityRating, LOW PERCENTAGE' do
+      describe '#is_va_service_connected' do
+        it 'returns false if veteran is paid for a disability with a low disability percentage' do
+          VCR.use_cassette('va_profile/disability/disabilityRating_200_low_disability') do
+            response = subject.is_va_service_connected
+
+            expect(response).to eq(false)
+          end
+        end
+      end
+
+      describe '#compensable_va_service_connected' do
+        it 'returns true if veteran is paid for a disability with a low disability percentage' do
+          VCR.use_cassette('va_profile/disability/disabilityRating_200_low_disability') do
+            response = subject.compensable_va_service_connected
+
+            expect(response).to eq(true)
+          end
+        end
+      end
+
+      describe '#va_compensation_type' do
+        it "returns 'lowDisability' when veteran is paid for a lowDisbility" do
+          VCR.use_cassette('va_profile/disability/disabilityRating_200_low_disability') do
+            response = subject.va_compensation_type
+
+            expect(response).to eq('lowDisability')
+          end
+        end
+      end
+    end
+
+    context 'military personnel service' do
+      describe '#sw_asia_combat' do
+        it 'returns if veteran was deployed to sw asia during gulf war' do
+          VCR.use_cassette('va_profile/military_personnel/post_read_service_history_200',
+                           match_requests_on: %i[method body]) do
+            expect(subject.sw_asia_combat).to eq(false)
+          end
+        end
+      end
+
+      describe '#discharge_type' do
+        it 'returns discharge type' do
+          VCR.use_cassette('va_profile/military_personnel/post_read_service_history_200',
+                           match_requests_on: %i[method body]) do
+            expect(subject.discharge_type).to eq('general')
+          end
+        end
+
+        it 'with an unknown character_of_discharge_code it returns nil' do
+          # rubocop:disable RSpec/SubjectStub
+          allow(subject).to receive(:latest_service_episode).and_return(
```

Comment:
Any reason you can't use a cassette for this case? If you can't, I still wouldn't stub subject, doing that is dangerously close to giving you a false positive.

Reply:
No reason! We have a cassette we can use for this.